#!/usr/bin/python3
def raise_exception_msg(message=""):
    """
    Raises a NameError exception with a custom message.
    Args:
        message (str): The message to be included in the exception.
    """
    raise NameError(message)
